app/src/main/java/com/am/yo_yo/app/YoYoService.java

```diff
@@ -139,8 +139,11 @@ private void startTest() {
             private int index = startCountDownInSecs;
 
             public void onTick(long millisUntilFinished) {
+              if(index>0){
+                // enables not to send last number command to tts as its done in on finish
```

Comment:
I had put this fix in just this morning and published a new release build.
https://github.com/AbhijithMadhav/yo-yo/commit/926144d25454ac41a584a65e4a70bcc809ad8e63

Reply:
but this is missed in start test function